src/storage-controller/src/lib.rs

```diff
@@ -196,6 +197,9 @@ pub struct Controller<T: Timestamp + Lattice + Codec64 + From<EpochMillis> + Tim
     /// Compaction commands to send during the next call to
     /// `StorageController::process`.
     pending_compaction_commands: Vec<PendingCompactionCommand<T>>,
+    /// Channel for table handle drops.
+    #[derivative(Debug = "ignore")]
+    pending_table_handle_drops: VecDeque<oneshot::Receiver<Vec<GlobalId>>>,
```

Comment:
Couldn't we use a channel for this? Then `pending_table_handle_drops` would become a `UnboundedReceiver<GlobalId>`, and the task in `drop_tables` would not need a fresh `tx, rx` pair but just a handle to this channel here and it sends to it.

Reply:
Good idea, just updated this to a channel.